site/src/store/slices/courseRequirementsSlice.ts

```diff
@@ -11,15 +11,20 @@ export interface MajorWithSpecialization {
 
 type ExpandedGroupsList = { [key: string]: boolean | undefined };
 
+export interface minorRequirements {
```

Comment:
Interface should be `PascalCase`, not `camelCase`
```suggestion
export interface MinorRequirements {
```